src/annotator/test/bucket-bar-test.js

```diff
@@ -113,5 +113,11 @@ describe('BucketBar', () => {
       bucketBar.update();
       assert.notCalled(window.requestAnimationFrame);
     });
+
+    it('deletes the bucketbar element after destroy method is called', () => {
+      const bucketBar = createBucketBar();
+      bucketBar.destroy();
+      assert.isNull(bucketBar.element.parentElement);
```

Comment:
I would do the assert from the test node rather than peaking into the bucketBar's member `element`. If we change it, then we can treat it more like a blackbox and have less code-specific tests.

`assert.isFalse(container.hasChildNodes())`

I noticed a previous test also reference .element, but possibly could be refactored to use `container`
OR  is `element` intended to be public?

Reply:
It is a good suggestion. Thank you very much.